openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -25,6 +25,7 @@ class CreateReview(plugin.Creator):
         "depth peeling",
         "alpha cut"
     ]
+    displayLights = ["default", "all", "selected", "active", "none"]
 
```

Comment:
This data should be shared between collector and extractor from a central location. Where would be the best place for it?

Reply:
@antirotor what do you think?

Reply:
hmm, maybe collector that can be used to collect stuff now processed by `extract_playblast` - like `collect_viewport`? It could put it under instace dictionary along with other related stuff.

Reply:
Ideally it should be available to the creator plugin as well, so maybe from a library?
Not sure whether OpenPype stores data in libraries?